src/client/views/test-view-iframe.html

```diff
@@ -0,0 +1,7 @@
+<html>
+
+<body class="ms-font-m">
+	<iframe src="https://localhost:3000/#/view/private-samples/excel/01-basics/basic-api-call" style="height:300px; width:600px;"></iframe>
```

Comment:
This should not contain localhost it should be relative to the root. <iframe src="/#/view/private-samples/excel/01-basics/basic-api-call"